lib/errorHandlers/index.ts

```diff
@@ -1,43 +1,35 @@
-// @ts-nocheck
-const { logger } = require('@hubspot/local-dev-lib/logger');
-const {
+import { logger } from '@hubspot/local-dev-lib/logger';
+import {
   isHubSpotHttpError,
-  isSystemError,
-  isFileSystemError,
   isValidationError,
-  isMissingScopeError,
-} = require('@hubspot/local-dev-lib/errors/index');
-const { shouldSuppressError } = require('./suppressError');
-const { i18n } = require('../lang');
-const util = require('util');
-const { isAxiosError } = require('axios');
+} from '@hubspot/local-dev-lib/errors/index';
+import { shouldSuppressError } from './suppressError';
+import { i18n } from '../lang';
+import util from 'util';
 
 const i18nKey = 'lib.errorHandlers.index';
 
-function logError(error, context = {}) {
+export function logError(error: unknown, context?: ApiErrorContext): void {
   debugError(error, context);
 
-  if (
-    shouldSuppressError(error, context) ||
-    shouldSuppressError(error, error.context)
-  ) {
+  if (shouldSuppressError(error, context)) {
     return;
   }
 
-  if (isHubSpotHttpError(error) && context) {
+  if (isHubSpotHttpError(error) && 'context' in error) {
+    if (shouldSuppressError(error, error.context)) {
+      return;
+    }
+  }
+
+  if (isHubSpotHttpError(error) && context instanceof ApiErrorContext) {
```

Comment:
This can just be  `isHubSpotHttpError(error) && context` since we know that if `context` exists, its of type `ApiErrorContext`

Reply:
https://github.com/HubSpot/hubspot-cli/pull/1251/commits/38daa6fc28f0896853743c7df3b3312284c1420d